[carry 13231 ] DOCS: Added support for arch s390x for Ubuntu, RHEL and SLES (#13288)
docker/docker.github.io: engine/install/rhel.md

>
> ```console
> $ sudo yum-config-manager --enable docker-ce-test
> ```
>
> You can disable the **nightly** or **test** repository by running the
> `yum-config-manager` command with the `--disable` flag. To re-enable it, use
> the `--enable` flag. The following command disables the **nightly** repository.
>
> ```console
> $ sudo yum-config-manager --disable docker-ce-nightly
> ```
>
> [Learn about **nightly** and **test** channels](index.md).

#### Install Docker Engine

1.  Install the _latest version_ of Docker Engine and containerd, or go to the next step to install a specific version:

    ```console
    $ sudo yum install docker-ce docker-ce-cli containerd.io
    ```

    If prompted to accept the GPG key, verify that the fingerprint matches
    `060A 61C5 1B55 8A7F 742B 77AA C52F EB6B 621E 9F35`, and if so, accept it.

    > Got multiple Docker repositories?
    >
    > If you have multiple Docker repositories enabled, installing
    > or updating without specifying a version in the `yum install` or
    > `yum update` command always installs the highest possible version,
    > which may not be appropriate for your stability needs.

>     Docker is installed but not started. The `docker` group is created, but no users are added to the group.

Reviewer: ```suggestion
This command installs Docker, but it doesn't start Docker. It also creates a Docker group, however, it doesn't add any users to the group by default. 
```

Author: The majority of this page is the same for all  the RPM installs; I kept it for now, but we can change it for all of them if you want (perhaps as a follow-up).

2.  To install a _specific version_ of Docker Engine, list the available versions
    in the repo, then select and install:

    a. List and sort the versions available in your repo. This example sorts
       results by version number, highest to lowest, and is truncated:

    ```console
    $ yum list docker-ce --showduplicates | sort -r

    docker-ce.s390x                3:20.10.7-3.el8                 docker-ce-stable
    ```

    The list returned depends on which repositories are enabled, and is specific
    to your version of RHEL (indicated by the `.el8` suffix in this example).

    b. Install a specific version by its fully qualified package name, which is
       the package name (`docker-ce`) plus the version string (2nd column)
       starting at the first colon (`:`), up to the first hyphen, separated by
       a hyphen (`-`). For example, `docker-ce-20.10.7`.

    ```console
    $ sudo yum install docker-ce-<VERSION_STRING> docker-ce-cli-<VERSION_STRING> containerd.io
    ```

>     Docker is installed but not started. The `docker` group is created, but no users are added to the group.

Reviewer: This command installs Docker, but it doesn't start Docker. It also creates a `docker` group, however, it doesn't add any users to the group by default.

3.  Start Docker.

    ```console
    $ sudo systemctl start docker
    ```

4.  Verify that Docker Engine is installed correctly by running the `hello-world`
    image.

    ```console
    $ sudo docker run hello-world
    ```

    This command downloads a test image and runs it in a container. When the
    container runs, it prints an informational message and exits.

> Docker Engine is installed and running. You need to use `sudo` to run Docker

Reviewer: ```suggestion
This installs and runs Docker Engine. Use `sudo` to run Docker
```

commands. Continue to [Linux postinstall](linux-postinstall.md) to allow
non-privileged users to run Docker commands and for other optional configuration
steps.

#### Upgrade Docker Engine

To upgrade Docker Engine, follow the [installation instructions](#install-using-the-repository),
choosing the new version you want to install.

### Install from a package

If you cannot use Docker's repository to install Docker, you can download the
`.rpm` file for your release and install it manually. You need to download
a new file each time you want to upgrade Docker Engine.

1.  Go to [{{ download-url-base }}/]({{ download-url-base }}/){: target="_blank" rel="noopener" class="_" }
    and choose your version of RHEL. Then browse to `s390x/stable/Packages/`
    and download the `.rpm` file for the Docker version you want to install.

>     > **Note**: To install a **nightly** or **test** (pre-release) package,

Reviewer: ```suggestion
    > **Note**
    >
    >  To install a **nightly** or **test** (pre-release) package,
```

    > change the word `stable` in the above URL to `nightly` or `test`.
    > [Learn about **nightly** and **test** channels](index.md).

2.  Install Docker Engine, changing the path below to the path where you downloaded
    the Docker package.

    ```console
    $ sudo yum install /path/to/package.rpm
    ```

    Docker is installed but not started. The `docker` group is created, but no
    users are added to the group.

3.  Start Docker.

    ```console
    $ sudo systemctl start docker
    ```

4.  Verify that Docker Engine is installed correctly by running the `hello-world`
    image.

    ```console
    $ sudo docker run hello-world
    ```

    This command downloads a test image and runs it in a container. When the

>     container runs, it prints an informational message and exits.

Reviewer: ```suggestion
    container runs, it prints a message and exits.
```

Docker Engine is installed and running. You need to use `sudo` to run Docker commands.
Continue to [Post-installation steps for Linux](linux-postinstall.md) to allow
non-privileged users to run Docker commands and for other optional configuration
steps.

#### Upgrade Docker Engine

To upgrade Docker Engine, download the newer package file and repeat the
[installation procedure](#install-from-a-package), using `yum -y upgrade`

> instead of `yum -y install`, and pointing to the new file.

Reviewer: ```suggestion
instead of `yum -y install`, and point to the new file.
```
